### ISSUE TITLE
Add  PA support for transpose of convection operator

### ISSUE THREAD

**Issue by artv3** [mfem/mfem#2838]
Transposing the convection integrator and using PA
```
   k->AddDomainIntegrator(new TransposeIntegrator(new ConvectionIntegrator(velocity, alpha)));                     
```
leads to the following error:
```
BilinearFormIntegrator::AddMultTransposePA(...)
   is not implemented for this class.
```

**Comment by tzanio** [comment]
See https://github.com/mfem/mfem/pull/2840

**Comment by tzanio** [comment]
Addressed in #2840.
